app/models/spree/back_in_stock_notification.rb

```diff
@@ -3,8 +3,8 @@
 class Spree::BackInStockNotification < ApplicationRecord
   paginates_per 50
 
-  # number of seconds between each email being sent (can be a fraction)
-  EMAIL_DELAY_FACTOR = 1
+  # number of seconds between each email being sent
+  EMAIL_DELAY_FACTOR = 0.5
```

Comment:
With everything being run in individual jobs now, I'm skeptical that adding the delay to the jobs is needed, up to you.

Reply:
Yes, I was thinking the same. I'll remove it now.